### PR TITLE
clarify starting test server

### PR DESCRIPTION
**Pull request by TheMagicNacho**
Documentation on how to start the test server felt unclear, added reference link to how to start build and start server.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @TheMagicNacho! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/TheMagicNacho/cesium/blob/doc-updates/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
    * Maintainers, [here's how to check](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CLAs/README.md) for a submitted CLA.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/doc-updates/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by TheMagicNacho** [comment]
1. I added myself to the CONTRIBUTORS.md file.
2. CLA is signed
3.  These are edits to documentation so no changes to the CHANGES.md is required. No changes or updates to tests either.

File: Documentation/Contributors/TestingGuide/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ All new code should have 100% code coverage and should pass all tests. Always ru
 
 The CesiumJS tests are written in JavaScript and use [Jasmine](http://jasmine.github.io/), a behavior-driven testing framework. Jasmine calls an individual test, e.g., a function with one or more assertions, a **spec** (however, the Cesium team usually still say "test"), and a group of related tests, e.g., all the tests for `Cartesian3`, a **suite**. Jasmine also calls an assertion, an **expectation**.
 
-When running CesiumJS locally, browse to [http://localhost:8080/](http://localhost:8080/) and there are several test options:
+When running CesiumJS locally, [start the local server](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/BuildGuide#build-the-code), browse to [http://localhost:8080/](http://localhost:8080/) and there are several test options:
```
Comment (ggetz) [inline Documentation/Contributors/TestingGuide/README.md:46]:
```suggestion
When running CesiumJS locally, [start the local server](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/BuildGuide#build-the-code) and browse to [http://localhost:8080/](http://localhost:8080/). There are several test options:
```

**Review by ggetz — COMMENTED** [review]
Thanks @TheMagicNacho ! I just a minor grammar suggestion.

**Comment by TheMagicNacho** [comment]
@ggetz thanks! I committed your edits.

**Comment by ggetz** [comment]
Thanks @TheMagicNacho !

**Pull request merged**